test/Parse/extractConstantsFromMembers_attr.swift

```diff
@@ -0,0 +1,4 @@
+// RUN: %target-typecheck-verify-swift -parse -enable-experimental-feature ExtractConstantsFromMembers
```

Comment:
Could you please add a small lit test to ensure we emit `attr_extractConstantsFromMembers_experimental` when not enabled with the opt-in flag?

Reply:
Good catch, I've added a test case for this